adr/ADR-32.md

```diff
@@ -223,15 +246,44 @@ The type for this is `io.nats.micro.v1.schema_response`.
     * Average processing_time is the total processing_time divided by the num_requests
     */
     average_processing_time: Nanos;
-    /**
-    * ISO Date string when the service started in UTC timezone
-    */
-    started: string
 }
 ```
 
 The type for this is `io.nats.micro.v1.stats_response`.
 
+## Adding groups and endpoints
+
+A service can be extended by adding additional groups and endpoints.
+
+### Groups
+
+A group serves as a common prefix to all endpoints registered in it.
+A group can be created using `addGroup(name)` method on a Service.
+Group name should be a valid NATS subject, but cannot contain `>`
```

Comment:
_but cannot end in `>`_

Reply:
Well, I suppose it is pretty much the same thing right? I mean, nats subject can only have `>` at the end, group cannot have it anywhere.